tests/bgp/bgp_helpers.py

```diff
@@ -30,22 +30,27 @@ def apply_bgp_config(duthost, template_name):
         duthost: DUT host object
         template_name: pathname of the bgp config on the DUT
     """
-    duthost.shell('docker cp {} {}'.format(template_name, DEFAULT_BGP_CONFIG))
-    restart_bgp(duthost)
-
+    duthost.docker_copy_to_all_asics('bgp', template_name, DEFAULT_BGP_CONFIG)
+    duthost.restart_bgp()
+    # Check if all BGPs are up
+    if duthost.is_multi_asic:
+        for asic_index in range(duthost.facts["num_asic"]):
+            docker_name = duthost.asic_instance(asic_index).get_docker_name("bgp")
+            pytest_assert(wait_until(100, 10, 0, duthost.is_service_fully_started, docker_name), "BGP not started.")
+    else:
+        pytest_assert(wait_until(100, 10, 0, duthost.is_service_fully_started, "bgp"), "BGP not started.") 
 
 def restart_bgp(duthost, asic_index=DEFAULT_ASIC_ID):
```

Comment:
in this file in some places we use `restart_bgp` and in some places we use `duthost.restart_bgp`. Can we use one function everywhere

Reply:
I have thought of this, is it ok to check is_fully_started inside multi_asic.py or need to make it in bgp_helpers.py each time?

Reply:
PR updated, can you please review gain?